pkg/querier/queryrange/codec.go

```diff
@@ -1194,7 +1183,7 @@ func (Codec) MergeResponse(responses ...queryrangebase.Response) (queryrangebase
 			for _, series := range lokiResult.Data {
 				// Use series hash as the key and reuse key
 				// buffer to avoid extra allocations.
-				key, keyBuffer = series.Hash(b, keyBuffer)
+				key = series.Hash(b)
```

Comment:
What's the reason for removing the `keyBuffer`? Or can you remove the comment above?

Reply:
There's no need for a buffer anymore as we can sort the slice of labels directly.